### PR TITLE
[upd] name of collider_box element and [add] volume_type and mouse_ju…

### PR DESCRIPTION
**Pull request by pokerverse**
…mp element properties

### CODE REVIEW

File: src/r/recordTypes/Element.ts [hunk 1]
```diff
@@ -268,4 +271,6 @@ export const elementPropertyDefaults: Record<ElementProperty, unknown> = {
   [ElementProperty.apply_env_map]: true,
   [ElementProperty.env_map_intensity]: 1,
   [ElementProperty.preload]: false,
+  [ElementProperty.volume_type]: "cube",
```
Comment (amitrajput1992) [inline src/r/recordTypes/Element.ts:274]: create an enum for volume type. export from `en` namespace

File: src/r/recordTypes/Element.ts [hunk 2]
```diff
@@ -268,4 +271,6 @@ export const elementPropertyDefaults: Record<ElementProperty, unknown> = {
...
+  [ElementProperty.mouse_jump]: true
```
Comment (amitrajput1992) [inline src/r/recordTypes/Element.ts:275]: also add a migration to convert all existing collider box elements to `box` type

**Review by amitrajput1992 — CHANGES_REQUESTED** [review]

File: src/r/recordTypes/Element.ts (hunk 1 above)
Comment (amitrajput1992) [inline src/r/recordTypes/Element.ts:274]: Use the enum you created here

File: src/migrations/project/r-migration-commands/m144_145_add_volume_type_default_value_to_collider_box_element.ts [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+    const allColliderBoxElements = projectF.getAllDeepChildrenWithFilter(RT.element, el => el.props.element_type === "collider_box");
+
+    for (const el of allColliderBoxElements) {
+      const elementF = r.element(el);
+      elementF.set(rtp.element.element_type, en.ElementType.collider_volume);
+      elementF.set(rtp.element.volume_type, en.VolumeTypes.cube);
+    }
+
+    projectF.set(rtp.project.version, 145);
```
Comment (amitrajput1992) [inline src/migrations/project/r-migration-commands/m144_145_add_volume_type_default_value_to_collider_box_element.ts:21]: Verify thé versioning from last couple of migrations. Easy to mess up versions

**Review by amitrajput1992 — CHANGES_REQUESTED** [review]

File: src/r/recordTypes/Element.ts (hunk 1 above)
Comment (pokerverse) [reply src/r/recordTypes/Element.ts:274]: When I did that, it gave me an error.

File: src/migrations/project/r-migration-commands/m144_145_add_volume_type_default_value_to_collider_box_element.ts (hunk 3 above)
Comment (pokerverse) [reply src/migrations/project/r-migration-commands/m144_145_add_volume_type_default_value_to_collider_box_element.ts:21]: Verified and checked

File: src/r/recordTypes/Element.ts (hunk 1 above)
Comment (pokerverse) [reply src/r/recordTypes/Element.ts:274]: image
This is the error I kept getting.

**Review by amitrajput1992 — APPROVED** [review]

**Pull request merged**